File: ApiAIDemo/ApiAI/ResultViewController/ResultViewController.m
//
//  ResultViewController.m
//  ApiAIDemo
//
//  Created by Kuragin Dmitriy on 26/01/15.
//  Copyright (c) 2015 Kuragin Dmitriy. All rights reserved.
//

#import "ResultViewController.h"

@interface ResultViewController ()

@end

@implementation ResultViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    _textView.text = [_response description];
}

- (IBAction)done:(id)sender
{
    [self dismissViewControllerAnimated:YES completion:NULL];
}

@end

File: ApiAIDemoSwift/ApiAIDemoSwift/ResultViewController/ResultViewController.swift
//
//  ResultViewController.swift
//  ApiAIDemoSwift
//
//  Created by Kuragin Dmitriy on 25/03/15.
//  Copyright (c) 2015 Kuragin Dmitriy. All rights reserved.
//

import UIKit

class ResultViewController: UIViewController {
    @IBOutlet fileprivate var textView: UITextView? = nil
    
    var response: AnyObject? = nil
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.textView?.text = self.response?.description
    }

    @IBAction func done(_ sender: UIBarButtonItem) {
        self.dismiss(animated: true, completion: nil)
    }
}
